### PR TITLE
Issue#88

### PR DESCRIPTION
**Pull request by excellencetechnologies**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ import { CategoryComponent } from '../components/categoryProducts/CategoryProduc
 import { FocusByEnterKey } from '../directives/focusByEnterKey/nextByEnter';
 import { Headers} from '../directives/headers/headers';
 import { SideMenu} from '../directives/sidemenu/sidemenu';
-
+import { EqualValidator } from '../directives/equal-validator/equalValidator';
```
Comment (excellencetechnologies) [inline src/app/app.module.ts]: rename /equal-validator/equalValidator  to /password/equalValidator

File: src/components/submitReview/SubmitReview.ts [hunk 2]
```diff
@@ -48,7 +48,9 @@ export class SubmitReview implements OnInit {
     onSelectRatting(rating, title) {
         this.selectedRating = rating;
     }
+
     submitReview() {
+     var num:any=1;
```
Comment (excellencetechnologies) [inline src/components/submitReview/SubmitReview.ts]: what is this?

File: src/directives/focusByEnterKey/nextByEnter.ts [hunk 3]
```diff
@@ -4,20 +4,22 @@ import { Directive, ElementRef, HostListener, Input } from '@angular/core';
...
    }
    @HostListener('keydown', ['$event']) onKeyDown(e) {
        if ((e.which == 13 || e.keyCode == 13)) {
-           e.preventDefault();
-           if (e.srcElement.nextElementSibling) {
-            e.srcElement.nextElementSibling.focus();
-            }
-           else{
-           }
+        if (this.id!=100) {
```
Comment (excellencetechnologies) [inline src/directives/focusByEnterKey/nextByEnter.ts]: why is 100 hard coded

File: src/directives/focusByEnterKey/nextByEnter.ts [hunk 4]
```diff
@@ -4,20 +4,22 @@ import { Directive, ElementRef, HostListener, Input } from '@angular/core';
...
+         e.preventDefault();
+         var ele= document.getElementById(this.id);
+           let child=ele.childNodes[0];
+           child.focus();
            return;
        }
-   }
+       else{
+       }
```
Comment (excellencetechnologies) [inline src/directives/focusByEnterKey/nextByEnter.ts]: format code

File: src/pages/changePassword/changePassword.ts [hunk 5]
```diff
@@ -1,48 +1,47 @@
...
-                secret: [this.secret],
-                access_token: [this.access_token]
-            });
-            this.changeActive = true;
+        this.user = {
+            password: '',
+            newPassword: '',
+            confirmPassword: ''
+        }
+         this._local.get('access_token').then((access_token: any) => {
```
Comment (excellencetechnologies) [inline src/pages/changePassword/changePassword.ts]: this has changed entirely by yuvraj

File: src/pages/changePassword/changePassword.ts [hunk 6]
```diff
@@ -1,48 +1,47 @@
...
+    }
+    changepassword(model: User, isValid: boolean) {
         this.spin = true;
-        this._changePwd.getPwd(value).then((res) => {
+        let data = { "password": model.password, "newPassword": model.newPassword, "secret": this.secret,access_token:this.access_token }
+        this._changePwd.getPwd(data).then((res) => {
             this.spin = false;
             this.response = res;
-            this._toast.toast(this.response.data,3000,"top");
+            this._toast.toast(this.response.data, 3000, "top");
```
Comment (excellencetechnologies) [inline src/pages/changePassword/changePassword.ts]: show toast in bottom always

File: src/pages/changePassword/changepassword.html [hunk 7]
```diff
@@ -9,19 +9,30 @@
...
+    <form #f="ngForm" id="form" novalidate (ngSubmit)="changepassword(f.value, f.valid)" >
+        <ion-item> 
             <ion-label floating>Old Password</ion-label>
-            <ion-input type="password"  formControlName="password" autocorrect autocomplete clearInput></ion-input>
+            <ion-input type="password" id="31" [onReturn]='32' name="password" [ngModel]="user.password" minlength="6" required reverse="true" #password="ngModel" autocorrect autocomplete clearInput></ion-input>
         </ion-item>
         <ion-item>
             <ion-label floating>New password</ion-label>
-            <ion-input type="password"  formControlName="newPassword" autocorrect autocomplete clearInput></ion-input>
+            <ion-input type="password" id="32" [onReturn]='34' name="newPassword"
```
Comment (excellencetechnologies) [inline src/pages/changePassword/changepassword.html]: instead of using 32, 34 as id
can we ur proper string

File: src/pages/register/register.ts [hunk 8]
```diff
@@ -56,6 +56,7 @@ export class RegisterPage implements OnInit {
                 this._local.set('expiry', this.data.data.expiry);
```
Comment (excellencetechnologies) [inline src/pages/register/register.ts]: local storage has fully been changed by yuvraj

File: src/app/app.module.ts (hunk 1 above)
Comment (upexcel) [inline src/app/app.module.ts]: done

File: src/directives/focusByEnterKey/nextByEnter.ts (hunk 3 above)
Comment (upexcel) [inline src/directives/focusByEnterKey/nextByEnter.ts]: fix

File: src/pages/changePassword/changePassword.ts (hunk 5 above)
Comment (upexcel) [inline src/pages/changePassword/changePassword.ts]: It is missed by yuvraj know it fixed

File: src/components/submitReview/SubmitReview.ts (hunk 2 above)
Comment (upexcel) [inline src/components/submitReview/SubmitReview.ts]: it is done by siddharth sir to remove error of comparision  number to string

**Pull request merged**